src/interfaces/IPeripheryPayments.sol

```diff
@@ -1,28 +1,28 @@
 // SPDX-License-Identifier: GPL-2.0-or-later
-pragma solidity >=0.7.5;
+pragma solidity 0.8.13;
```

Comment:
Btw, I'm happy with `^version` because we lock the version in in `foundry.toml`. This makes it easier to bump new versions. That said, no need to change just for future files/reference. Only time to change version is if 0.8.13 has a bug, then it can be good to raise to the version where the bug is fixed.

Btw 0.8.13 does have bugs, so make a task to raise versions across the board, maybe just move all files to `^latest` (in a separate PR).